draft-ietf-quic-recovery.md

```diff
@@ -1175,7 +1175,7 @@ GetEarliestLossTime():
   space = Initial
   for pn_space in [ Handshake, ApplicationData ]:
     if (loss_time[pn_space] != 0 &&
-        (time == 0 || loss_time[pn_space] < time)):
+        (time == 0 or loss_time[pn_space] < time)):
```

Comment:
so && for and and "or" for || ?

Reply:
Prior to this PR, most of the pseudocode was && and || I realize, so I'm going back to that.